red_october/models/res_company.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+
+    _inherit = 'res.company'
+
+    default_red_october_id = fields.Many2one(
+        string='Default Crypto Vault',
+        comodel_name='red.october.vault',
+        store=True,
+        compute="_compute_default_red_october_id",
```

Comment:
we have a different convention for that: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#models